BrickTests/Shared/ViewModelSpec.swift

```diff
@@ -3,11 +3,11 @@ import Quick
 import Nimble
 import Fakery
 
-class ViewModelSpec: QuickSpec {
+class ItemSpec: QuickSpec {
 
   override func spec() {
-    describe("ViewModel") {
-      var viewModel: ViewModel!
+    describe("Item") {
+      var Item: Item!
```

Comment:
Maybe it should build the tests when calling `build` instead of just when the tests run. That would avoid these kind of things.